.gitignore

```diff
@@ -6,3 +6,4 @@
 /.project
 /.pydevproject
 /.settings
+*~
```

Comment:
This is for ignoring emacs backup files. Emacs is my IDE of choice. When a file is modified then it puts it old version with extension ~, like README.md~